### PR TITLE
18.10.1 - Clear release notes

### PR DESCRIPTION
**Pull request by Itay4**

### CODE REVIEW

File: Tests/scripts/server_get_artifact.sh [hunk 1]
```diff
@@ -18,13 +18,13 @@ if [[ "$ARTIFACT_BUILD_NUM" = "" ]]; then
     exit 1
 fi
 
-SERVER_DOWNLOAD_LINK=$(curl -s -H "$ACCEPT_TYPE" ${SERVER_API_URI}/${ARTIFACT_BUILD_NUM}/artifacts?${TOKEN_ATTR} | jq '.[].url' -r | grep demistoserver | grep /0/)
```
Comment (asafshen) [inline Tests/scripts/server_get_artifact.sh]: what are the purpose of this change?

Comment (Itay4) [reply Tests/scripts/server_get_artifact.sh]: Cause of @ronykoz changes in another branch and I don't want to merge from master

Comment (yaakovi) [reply Tests/scripts/server_get_artifact.sh]: server installer download returned more then one result with those properties (for some reason) in build 32322

Comment (asafshen) [reply Tests/scripts/server_get_artifact.sh]: image

jus saw this revert
cool

**Pull request merged**